### PR TITLE
Add new hyperkube versions

### PR DESCRIPTION
**Pull request by calvix**
Add new hyperkube versions

### CODE REVIEW

File: images.go [hunk 1]
```diff
@@ -106,6 +106,14 @@ var Images = []Image{
...
 				Tag: "v1.6.4_coreos.0",
 			},
+			Tag{
+				Sha: "b26b7a598382e9db0623dd999f5506746a076b866f12188010f6c07fef26e4da",
+				Tag: "v1.6.7_coreos.0",
+			},
+			Tag{
+				Sha: "cc57bd170e562ab699b6dd2244f2683018214c8228591dc641cffce0f0f92037",
+				Tag: "v1.7.1_coreos.0",
+			},
```
Comment (xh3b4sd) [inline images.go]: Is there maybe some rule or guideline to only support the last two versions? Should we delete the oldest one now?

Comment (calvix) [inline images.go]: I have no idea how we wanna do this, @JosephSalisbury ?

Comment (JosephSalisbury) [inline images.go]: No strong opinion at the moment.

**Comment by calvix** [comment]
ok we can decide later

**Comment by JosephSalisbury** [comment]
See https://github.com/giantswarm/giantswarm/issues/1665 for tracking decision on retention for retagging images

**Pull request merged**